components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/IdentityEventConstants.java

```diff
@@ -141,6 +141,8 @@ private Event(){}
         public static final String POST_GET_ROLE_LIST_OF_USERS_WITH_ID = "POST_GET_ROLE_LIST_OF_USERS_WITH_ID";
         public static final String PRE_SET_CHALLENGE_QUESTION_ANSWERS = "PRE_SET_CHALLENGE_QUESTION_ANSWERS";
         public static final String POST_SET_CHALLENGE_QUESTION_ANSWERS = "POST_SET_CHALLENGE_QUESTION_ANSWERS";
+        public static final String PRE_SELF_SIGNUP = "PRE_SELF_SIGNUP";
```

Comment:
Can you check the possibility of supporting the following events? 

PRE_SELF_SIGNUP_REGISTER
POST_SELF_SIGNUP_REGISTER
PRE_SELF_SIGNUP_CONFIRM
POST_SELF_SIGNUP_CONFIRM

Reply:
Done with e2b909af475805b23c7f7ee05094ec5f227101ae